lib/services/computerVision/package.json

```diff
@@ -13,7 +13,7 @@
   "license": "MIT",
   "main": "./lib/computerVisionClient.js",
   "types": "./lib/computerVisionClient.d.ts",
-  "homepage": "https://github.com/azure/azure-sdk-for-node/tree/master/lib/services/computerVision",
```

Comment:
This isn't correct. Please regenerate with the latest version of autorest.nodejs.